<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ServerOptions</class>
 <widget class="QWidget" name="ServerOptions">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>740</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QComboBox" name="mapSelect">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>10</y>
     <width>221</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="startServer">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>570</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Start Server</string>
   </property>
  </widget>
  <widget class="QPushButton" name="backToMenu">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>570</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Back to Menu</string>
   </property>
  </widget>
  <widget class="QLabel" name="mapLabel">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>10</y>
     <width>85</width>
     <height>25</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>Map Size:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QListWidget" name="civList">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>371</width>
     <height>441</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="civLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>100</width>
     <height>25</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>Civilizations</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QComboBox" name="civSelect">
   <property name="geometry">
    <rect>
     <x>399</x>
     <y>420</y>
     <width>331</width>
     <height>33</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="maxVisibleItems">
    <number>4</number>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="leaderPortrait">
   <property name="geometry">
    <rect>
     <x>399</x>
     <y>110</y>
     <width>331</width>
     <height>301</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>331</width>
     <height>301</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>331</width>
     <height>301</height>
    </size>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="addCivPB">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>510</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QPushButton" name="removeCivPB">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>510</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="humanRB">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>60</y>
     <width>82</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Human</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="aiRB">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>60</y>
     <width>82</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>AI</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="IPEntry">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>470</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="IPLabel">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>470</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>Server IP:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
